### ISSUE TITLE
Some question about chamfer distance calculation

### ISSUE THREAD

**Issue by Excuse-mimimi** [iccv19/graphx-conv#13]
In **network.py line 21** ，the function normalized_chamfer_loss(), when the reduce == 'mean', it return loss*3000. ,why multiply 3000？

` return loss if reduce == 'sum' else loss * 3000.`

Then， ithe chamfer distance loss calculation in the **test.py line 57** confuse me , because the normalized_chamfer_loss().
Did it means the chamfer distance multiply 1000? I know I have made some mistakes.Please help me.
```
loss = sum([normalized_chamfer_loss(pred[None], gt[None]) for pred, gt in zip(pred_pc, gt_pc)]) / (
                            3. * len(gt_pc))
```
Look forward to your reply.

**Comment by justanhduc** [comment]
Hi @Excuse-mimimi. This is some heuristic scaling that I inherit from Pixel2Mesh. The data provided by Pixel2Mesh is heuristically scaled to a certain size, and the multiplication is to avoid having too many zeros in the loss. This is mentioned in the paper (probably the pixel2mesh paper as well).
